typescript/formsApps.d.ts

```diff
@@ -69,13 +69,41 @@ export type FormsAppScheduledTasksGroupMenuItem = FormsAppBaseMenuItem & {
   isDefault: boolean
 }
 
+export type FormsAppCPHCMSContentMenuItemListDisplayAttributeMeta = {
+  type: 'CP_HCMS_CONTENT_META'
+  attribute: 'meta.created' | 'meta.lastModified' | 'meta.id'
+}
+export type FormsAppCPHCMSContentMenuItemListDisplayAttributeFormElement = {
+  type: 'FORM_ELEMENT'
+  formElementId: string
+}
+export type FormsAppCPHCMSContentMenuItemListDisplayAttribute =
+  | FormsAppCPHCMSContentMenuItemListDisplayAttributeMeta
+  | FormsAppCPHCMSContentMenuItemListDisplayAttributeFormElement
```

Comment:
I'm going to approve it, but if there is another name for these that can reduce the length of the name, that would be good to do, feels a bit too long imo